### ISSUE TITLE
MUC Trouble

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#256]
```
===  IMPORTANT - READ THIS!  ===
An app log will help us greatly to analyze the problem. So ALWAYS generate
an AppLog, even if it does not show anything (which will tell use that we
have to improve the log part ;-) ).
http://code.google.com/p/gtalksms/wiki/GenerateAppLog
You can remove this message from the issue report if you have read it.
================================

What steps will reproduce the problem?
tried a couple of different MUC chat rooms
 let me k now what else to do to troubleshot the problem
What is the expected output?

muc room opens 

What do you see instead?
ncoming SMS: Unable to send conference room configuration form. Ex: 
not-acceptable(406)

Additional informations:
Phone model: Razr

```

Original issue reported on code.google.com by `tedt...@gmail.com` on 21 Feb 2012 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
Have you read the section "IMPORTANT - READ THIS!"? Please add an App Log.
```

Original comment by `fschm...@gmail.com` on 23 Feb 2012 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
I think this is the current issue originally reported in Issue 247: 
http://code.google.com/p/gtalksms/issues/detail?id=247&colspec=ID%20Stars%20Type
%20Status%20Priority%20Milestone%20Owner%20Summary
```

Original comment by `rondepas...@gmail.com` on 23 Feb 2012 at 8:19

**Comment by GoogleCodeExporter** [comment]
```
Set forceMucServer default value to true with conference.jwchat.org in order to 
have the fresh install configuration working.
```

Original comment by `Florent....@gmail.com` on 1 Apr 2012 at 1:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 3.5
```

Original comment by `Florent....@gmail.com` on 2 Apr 2012 at 9:38
- Changed state: **Deployed**
